azure-pipelines.sds.pr-release.yml

```diff
@@ -39,24 +39,18 @@ stages:
         steps:
           - checkout: self
 
-          - powershell: |
-              $ProjectKey = "$(Build.Repository.Name)" -ireplace "hmcts\/"
-              Write-Output ("Sonar Cloud project key: {0}" -f $ProjectKey)
-              Write-Host "##vso[task.setvariable variable=ProjectKey;]$ProjectKey"
-            displayName: Set Sonar Cloud Project Key
-
           - bash: |
               sed -i "s|</configuration>|<packageSourceCredentials><vh-packages><add key=\"Username\" value=\"PAT\" /><add key=\"ClearTextPassword\" value=\"$(System.AccessToken)\" /></vh-packages></packageSourceCredentials></configuration>|" nuget.sds.config
             displayName: Add NuGet Feed Authentication
             workingDirectory: ${{ variables.appName }}
 
           - task: DockerCompose@0
-            displayName: Runs tests
+            displayName: Run unit & integration tests
```

Comment:
Could you change this to title case please to follow all other display names

Run Unit & Integration Tests

Reply:
Could you also ensure all of your displayName are title case please.